lpa-submissions-e2e-tests/cypress/integration/pageFooter.feature

```diff
@@ -0,0 +1,16 @@
+As a LPA Planning officer
+I want to access the Planning Inspectorates policies
+So that I can see what they do with personal information when I’m using of the the Planning Inspectorate’s services
+
+Scenario Outline: AC-01 LPA taken to correct page when accessed links in the footer
+  Given LPA planning officer accesses the LPA Questionnaire
+  When they click on the '<links>' in the footer
+  Then the '<page>' correct page should be displayed
+  And the page title should be '<title>'
+  Examples:
+  |links             |page                                                                    |title|
```

Comment:
Minor thing but formatting seems a bit off for the file - perhaps run it through the vscode formatter?

Reply:
I will do that as part of step definition